scp/Chart.yaml

```diff
@@ -1,20 +1,58 @@
 apiVersion: v2
 name: scp
-description: A Helm chart for Kubernetes
+description: Self Hosted Data Protection Platform
 type: application
-version: 0.1.0
-appVersion: 0.1.1
+version: 0.1.1
+appVersion: 0.1.2
 
-dependencies:
-  - name: backend
-    repository: oci://ghcr.io/opentdf/charts
-    version: '0.0.0-sha-03c7ccc'
-    condition: backend.enabled
+otdfVersion: &otdfVersion "0.0.0-sha-bd7a8d2"
+otdfRepo: &otdfRepo "oci://ghcr.io/opentdf/charts"
 
+dependencies:
   - name: abacus
```

Comment:
no abacus in self-hosted

Reply:
Near term we need abacus AFAIK,  you can current disable with: abacus.enabled